config/locales/en.yml

```diff
@@ -1250,6 +1250,7 @@ en:
     submissions:
       pipeline_selection:
         title: Workflow Selection
+        loading: Workflow submission is being generated, this can take a litte bit of time.
```

Comment:
**nitpick (blocking):** Could this possibly be changed to `Preparing workflow submission for X samples, this may take a bit of time.`

And then insert the number of samples in their like @ChrisHuynh333 did in the create data export modal.

Reply:
Actually lets change the text to `Preparing workflow arguments for X samples, this may take a bit of time.` since it is actually preparing the arguments for the submission and not preparing the actual submission

Reply:
Updated in adc713a854b518ad1e235099aaae8ab43568d114